unstructured/partition/pdf_image/ocr.py

```diff
@@ -77,7 +81,10 @@ def process_data_with_ocr(
             ocr_mode=ocr_mode,
             pdf_image_dpi=pdf_image_dpi,
         )
-        return merged_layouts
+    finally:
+        os.remove(file_name)
```

Comment:
worth verifying the file exists before removing (though of course it will exist in the happy path), or catching FileNotFoundError?

Reply:
```suggestion
        if os.path.isfile(file_name):
            os.remove(file_name)
```